src/SUMMARY.md

```diff
@@ -20,7 +20,9 @@
 - [The Rustc Driver](./rustc-driver.md)
     - [Rustdoc](./rustdoc.md)
 - [Queries: demand-driven compilation](./query.md)
+    - [The Query Evaluation Model in Detail](./queries/query-evaluation-model-in-detail.md)
     - [Incremental compilation](./incremental-compilation.md)
```

Comment:
Perhaps move the `Incremental Compilation` chapter to the `queries/` directory too?